test/pom.xml

```diff
@@ -154,5 +154,15 @@
 				</dependency>
 		 </dependencies>
 		</profile>
+		<profile>
+			<id>proprietary</id>
+			<dependencies>
+				<dependency>
+					<groupId>com.ing</groupId>
+					<artifactId>ifsa-jms-xa</artifactId>
+					<scope>runtime</scope>
+				</dependency>
+			</dependencies>
+		</profile>
```

Comment:
Er is een proprietary profiel dat alle proprietary modules aanzet, een van die modules is IFSA, daar komt deze dependency uit. Waarom heb je deze hier los ingezet?

Reply:
Deze dependency ifsa-jms-xa staat in de pom.xml van ons WeAreFrank! Ifsa project. Maar hij staat daar met scope "provided". Daardoor werd deze .jar niet opgenomen in de Ibis4Test war. Onze IbisDoc code produceert daardoor een exception als hij voor een van onze IfsaProviderListener classes "Class.forName()" doet. Ik denk dat er Aspect Oriented Programming is gebruikt en dat we Ladybug aanroepen bij het laden van IfsaProviderListener. Het betreffende advice zou dan proberen een Ifsa class te laden, resulterend in de exception.

Deze fix verandert de scope van de Ifsa dependency van "provided" naar "runtime", waardoor hij in de ibis4test .war komt. Door deze noemen we de Ifsa dependency op verschillende plaaten: hier in de Ibis4Test pom en in ons Ifsa project. We willen de versie "22.10.055" niet twee keer noemen, vandaar dat ik de dependencyManagement van de parent pom heb aangepast.